torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -39,10 +39,13 @@ def _xyxy_to_cxcywh(xyxy: torch.Tensor) -> torch.Tensor:
 
 
 def convert_bounding_box_format(
-    bounding_box: torch.Tensor, *, old_format: BoundingBoxFormat, new_format: BoundingBoxFormat
+    bounding_box: torch.Tensor, old_format: BoundingBoxFormat, new_format: BoundingBoxFormat, copy: bool = False
```

Comment:
Implementing padding for bounding boxes, I realized we should not always copy here. Given that we don't perform an inplace operation, pass through can avoid unnecessary copies. A (future) common idiom is to convert a bounding box into a specific format, perform some operations on it and convert it back. If we set `copy=True` on the first conversion call, we can potentially avoid another copy on the reverse conversion.

Reply:
Sounds good but I would prefer to be on the safe side and have the default `copy=True`. Then we should turn it off explicitly when "it's OK, I know what I'm doing" situations. This is similar to how `inplace` works in PyTorch. We should also consider renaming this to `inplace` to be consistent with the names.

Reply:
> Sounds good but I would prefer to be on the safe side and have the default `copy=True`. Then we should turn it off explicitly when "it's OK, I know what I'm doing" situations.

:+1:

> This is similar to how `inplace` works in PyTorch. We should also consider renaming this to `inplace` to be consistent with the names.

Note that in the current state this is not a kernel that should have an `inplace` parameter, because it can be a no-op. So a comparison to something like `torch.Tensor.to` is better, which also just returns the input in case nothing needs to be done. 

```py
In [4]: t = torch.empty((), dtype=torch.float)
In [5]: t.to(torch.float) is t
Out[5]: True
In [6]: t.to(torch.int) is t
Out[6]: False
In [7]: t.to(torch.float, copy=True) is t
Out[7]: False
```

Reply:
We had a quick offline discussion about this. The difference between these two is as follows:
- If an op has a `copy` parameter it might be an actual no-op if `copy=False`. With `copy=True` or if any operation is performed, the user gets a new tensor. The old tensor is never changed inplace.
- If an op has an `inplace` parameter, it will always perform an operation. Whether this operation is performed inplace or on a new tensor is controlled by the flag.

Since the `convert_*` kernels fall in the first category, we are keeping the `copy=True` parameter.